tests/chainer_tests/functions_tests/loss_tests/test_triplet.py

```diff
@@ -30,6 +30,9 @@ def setUp(self):
         else:
             gy_shape = (self.batchsize,)
```

Comment:
Would you remove this unnecessary parentheses?

Reply:
Fixed.